pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -271,15 +272,14 @@ def determine_ddp_device_ids(self):
         return [self.root_device.index]
 
     def __transfer_distrib_spawn_state_on_fit_end(self, trainer: "pl.Trainer", results: Any) -> None:
+        rank_zero_info("Transferring results back to main process ...")
```

Comment:
Did you consider making this debug?

Reply:
I don't see a big difference. Could be any of the two. Definitely not a warning though.

Reply:
Should all spawn users see this? IMO no

Reply:
Let's make it debug. I got some complaints about it.

Reply:
Fine! 

#10864